### ISSUE TITLE
Coords, Sigma_y, and Sigma_x: which one is for horizontal and for vertical?

### ISSUE THREAD

**Issue by ChenWenxuan2021** [ChrisDud0257/GSASR#5]
I would like to thank the authors a lot for sharing the source codes of GSASR. 
However, I have met a problem to understand the following part:

`sigmas = torch.cat([sigma_y/step_size*2/(sr_size[1] - 1), sigma_x/step_size*2/(sr_size[0] - 1),  rho], dim=-1).contiguous()  # (gs num, 3)
 coords[:, 0] = (coords[:, 0] + 1 - 1/sr_size[1]) * sr_size[1] / (sr_size[1] - 1) - 1.0
 coords[:, 1] = (coords[:, 1] + 1 - 1/sr_size[0]) * sr_size[0] / (sr_size[0] - 1) - 1.0`

(./TrainTestGSASR/basicsr/utils/gaussian_splatting.py, line 88-90)

Generally, sigma_y and sigma_x are expected to represent the covariances of vertical and horizontal directions, respectively. However,  herein sigma_y is divided by sr_size[1], and sigma_x is divided by sr_size[0].
Moreover, are coord[:, 0] and coord[:, 1] for horizontal and for vertical respectively?

Looking forward to your response. Thank you very much!
